eclipse-collections-code-generator/src/main/resources/impl/list/immutable/immutablePrimitiveArrayList.stg

```diff
@@ -724,6 +726,12 @@ final class Immutable<name>ArrayList
             return next;
         }
     }
+
+    @Override
+    public RichIterable\<PrimitiveIterable> chunk(int size)
+    {
+        throw new UnsupportedOperationException(this.getClass().getSimpleName() + ".chunk() not implemented yet");
```

Comment:
Why is this one unsupported? Doesn't it inherit an implementation from an abstract superclass?

Reply:
This should have an appropriate implementation provided.